File: fetch_bundles.py
import urllib.request
import re
import os
import json

def fetch_bundles():
    js_url = 'https://champacademy.asia/assets/index-Cp5r5y2k.js'
    css_url = 'https://champacademy.asia/assets/index-CeWfe1qj.css'
    
    print('Fetching JS bundle...')
    with urllib.request.urlopen(js_url) as response:
        js_text = response.read().decode('utf-8')
    with open('index.js', 'w', encoding='utf-8') as f:
        f.write(js_text)
        
    print('Fetching CSS bundle...')
    with urllib.request.urlopen(css_url) as response:
        css_text = response.read().decode('utf-8')
    with open('index.css', 'w', encoding='utf-8') as f:
        f.write(css_text)
        
    print('Saved bundles index.js and index.css')

if __name__ == '__main__':
    fetch_bundles()

File: fetch_bundles.js
import fetch from 'node-fetch';
import fs from 'fs';
import path from 'path';

async function main() {
  const jsUrl = 'https://champacademy.asia/assets/index-Cp5r5y2k.js';
  const cssUrl = 'https://champacademy.asia/assets/index-CeWfe1qj.css';
  
  console.log('Fetching JS bundle...');
  const jsRes = await fetch(jsUrl);
  const jsText = await jsRes.text();
  fs.writeFileSync('index.js', jsText);
  
  console.log('Fetching CSS bundle...');
  const cssRes = await fetch(cssUrl);
  const cssText = await cssRes.text();
  fs.writeFileSync('index.css', cssText);
  
  console.log('Saved bundles.');
}

main().catch(console.error);
